### ISSUE TITLE
Build failure due to missing ukf-mda jar/pom files

### ISSUE THREAD

**Issue by xenophonf** [iay/mdq-server#37]
Maven returned the following error:

```
[ERROR] Failed to execute goal on project mdq-server: Could not resolve dependencies for project uk.org.iay.mdq:mdq-server:jar:0.0.1-SNAPSHOT: Could not find artifact uk.org.ukfederation:ukf-mda:jar:0.9.0 in shib-release (https://build.shibboleth.net/nexus/content/groups/public) -> [Help 1]
```
As a workaround I downloaded the missing files from https://github.com/ukf/ukf-mda/releases/download/v0.9.0/ukf-mda-0.9.0.jar (saved as `~/.m2/repository/uk/org/ukfederation/ukf-mda/0.9.0/ukf-mda-0.9.0.jar`) and https://raw.githubusercontent.com/ukf/ukf-mda/v0.9.0/pom.xml (saved as `~/.m2/repository/uk/org/ukfederation/ukf-mda/0.9.0/ukf-mda-0.9.0.pom`).

(FreeBSD/amd64 10.2-RELEASE, openjdk8-8.66.17_3, maven31-3.1.1_1)

**Comment by iay** [comment]
Good point. I normally build those myself locally as there is no Maven repository for the UKf projects. Until I get round to setting one up, you can get the artefacts directly from <https://github.com/ukf/ukf-mda/releases>.

**Comment by iay** [comment]
Resolved by 69af44592f422d6aff35e1671f9dc17c8d565304 (and a stack of infrastructure work, I have my Nexus repository manager back up and running again).
